services/movie_session.py

```
@@ -0,0 +1,61 @@
+from datetime import datetime
+from typing import Optional
+
+from django.db.models import QuerySet
+from django.shortcuts import get_object_or_404
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    try:
+        movie_session = MovieSession.objects.create(
+            show_time=movie_show_time,
+            movie_id=movie_id,
+            cinema_hall_id=cinema_hall_id
+        )
+        return movie_session
+    except Exception as e:
+        return e

```

u don't need to return the exception, just print some info